src/wp-admin/css/about.css

```diff
@@ -411,6 +411,10 @@
 	color: inherit;
 }
 
+.about__container :is(h1, h2, h3, h4):lang(ja) {
+	text-wrap: unset;
+}
+
```

Comment:
I mentioned in [the core ticket](https://core.trac.wordpress.org/ticket/60892#comment:15) that this could be adjusted in Korean as well, here's that suggestion:

```suggestion

.about__container :is(h1, h2, h3, h4):lang(ko) {
	word-break: keep-all;
}
```

This splits the string at the space between `에 오신`, rather than interrupting `오신`.

| Before | After |
|---|---|
| <img width="516" alt="Screenshot 2024-08-26 at 12 58 51 PM" src="https://github.com/user-attachments/assets/18e38ebb-5443-4d01-8ccd-765c1ba26295"> | <img width="532" alt="Screenshot 2024-08-26 at 12 57 01 PM" src="https://github.com/user-attachments/assets/192288e3-a03a-4e81-81ab-b8c3cd487b3e"> |